packages/kbn-test/src/junit_report_path.ts

```diff
@@ -17,14 +17,13 @@
  * under the License.
  */
 
-import { resolve } from 'path';
+import glob from 'glob';
+import { resolve, join } from 'path';
 import { CI_PARALLEL_PROCESS_PREFIX } from './ci_parallel_process_prefix';
 
 export function makeJunitReportPath(rootDirectory: string, reportName: string) {
```

Comment:
Is this method only used to find junit report paths? Maybe we should rename it as such because the current implementation seems it would fail if it was used for its original intended purpose of creating a report path for things to write to.